### ISSUE TITLE
loadkey Uk defaults to lang ukraine

### ISSUE THREAD

**Issue by ameeno** [MatMoul/archfi#60]
Hi, I believe UK keymap is British.
But the default language selected is Ukraine. Is that normal?

**Comment by MatMoul** [comment]
Yes and No, I've tried to preselect the nearest language from the keyboard selection but it seem that don't work for all...

**Comment by FredBezies** [comment]
Using localectl list-keymaps I noticed that ukranian keyboard and russian one are kinda the same.

```
localectl list-keymaps | grep uk                  
atari-uk-falcon
dvorak-uk
mac-uk
sunt5-uk
sunt6-uk
uk
```

Looking at russian output:

```
localectl list-keymaps | grep ru
dvorak-ru
ru
ru-cp1251
ru-ms
ru-yawerty
ru1
ru2
ru3
ru4
ru_win
ruwin_alt-CP1251
ruwin_alt-KOI8-R
ruwin_alt-UTF-8
ruwin_alt_sh-UTF-8
ruwin_cplk-CP1251
ruwin_cplk-KOI8-R
ruwin_cplk-UTF-8
ruwin_ct_sh-CP1251
ruwin_ct_sh-KOI8-R
ruwin_ct_sh-UTF-8
ruwin_ctrl-CP1251
ruwin_ctrl-KOI8-R
ruwin_ctrl-UTF-8
sunt5-ru
```

Hope it helps!

**Comment by MatMoul** [comment]
@FredBezies Yeah, it seem it's a good approach
```
localectl list-keymaps | grep fr_CH
fr_CH
fr_CH-latin1
mac-fr_CH-latin1
```
It seems good

**Comment by MatMoul** [comment]
@ameeno It seems that I did not understand the question, just looking for the keyboard layout...
@FredBezies looking at "localectl", I noticed that I used it in the past (It's commented in the code) but it was not available for a while anymore ...
It seems like he's back ...

**Comment by ameeno** [comment]
Hi, this is a bug report: whet I select UK as the keymap when installing your script.
UK - is for United Kingdom British I am sure

At the locale-gen section, Ukraine is uncommented, the default should be en_GB.utf8

**Comment by MatMoul** [comment]
@ameeno My first answer was the good ...
I just tested and I noticed the issue ...
Now, we'll have to find a way to map "uk" with "en_GB" ....
If possible with a generic solution ...

**Comment by x0Bloot** [comment]
btw layout for ukrainian is "ua" normally

**Comment by linuxgirl22** [comment]
Just to feedback that I tried to install archfi recently; the first install ended up in Ukrainian, as auto-selection for UK install was uk_UA. On the second install, I chose en_GB, which changed nearly everything to English. Home's folder titles were in Ukrainian though.
